app/views/applications/application.scala.html

```diff
@@ -97,12 +97,10 @@ <h3>Scopes</h3>
     }
 }
 
-@renderDeveloperUrl(checkInformation: CheckInformation) = {
-        @defining(DateTimeFormat.longDate) { dateFormatter =>
-        @defining(checkInformation.termsOfUseAgreements.filter(terms => terms.version == "1.0").head.emailAddress) { developerEmailAddress =>
-           Agreed by <a href="@{routes.DevelopersController.developerPage(developerEmailAddress)}">@developerEmailAddress</a> on @dateFormatter.print(checkInformation.termsOfUseAgreements.filter(terms => terms.version == "1.0").head.timeStamp)
-        }
-    }
+@renderTermsOfUseInfo(termsOfUseAgreement: TermsOfUseAgreement) = {
+    v@termsOfUseAgreement.version agreed by
+    <a href="@{routes.DevelopersController.developerPage(termsOfUseAgreement.emailAddress)}">@termsOfUseAgreement.emailAddress</a>
+    on @DateTimeFormat.longDate.print(termsOfUseAgreement.timeStamp)
```

Comment:
Change to use similar date time formatter as you are doing in third party dev frontend so that it explicitly outputs in 25 Jun 2018 format